package.json

```diff
@@ -69,6 +72,7 @@
   "scripts": {
     "production": "gulp build-production",
     "gulp": "gulp",
-    "development": "gulp build-development"
+    "development": "gulp build-development",
+    "lint": "eslint ."
```

Comment:
You may want to have this do `gulp lint` - because just running `eslint .` won't pick up the changes you made to include that folder of js files with html extensions.